src/Nri/Ui/Carousel/V1.elm

```diff
@@ -0,0 +1,94 @@
+module Nri.Ui.Carousel.V1 exposing (..)
+
+import Css exposing (..)
+import Html.Styled as Html exposing (Html)
+import Nri.Ui
+import Nri.Ui.Tooltip.V3 as Tooltip
+import TabsInternal.V2 as TabsInternal
+
+
+type Tab id msg
+    = Tab (TabsInternal.Tab id msg)
+
+
+type Attribute id msg
+    = Attribute (TabsInternal.Tab id msg -> TabsInternal.Tab id msg)
+
+
+type TabPosition
+    = Before
+    | After
+
+
+{-| Tooltip defaults: `[Tooltip.smallPadding, Tooltip.onBottom, Tooltip.fitToContent]`
+-}
+withTooltip : List (Tooltip.Attribute msg) -> Attribute id msg
+withTooltip attributes =
+    Attribute (\tab -> { tab | tabTooltip = attributes })
+
+
+{-| Makes it so that the tab can't be clicked or focused via keyboard navigation
+-}
+disabled : Bool -> Attribute id msg
+disabled isDisabled =
+    Attribute (\tab -> { tab | disabled = isDisabled })
+
+
+{-| -}
+slideHtml : Html msg -> Attribute id msg
+slideHtml content =
+    Attribute (\tab -> { tab | panelView = content })
+
+
+labelledBy : String -> Attribute id msg
+labelledBy labelledById =
+    Attribute (\tab -> { tab | labelledBy = Just labelledById })
+
+
+tabHtml : Html Never -> Attribute id msg
+tabHtml content =
+    Attribute (\tab -> { tab | tabView = [ Html.map never content ] })
+
+
+{-| -}
+buildTab : { id : id, idString : String } -> List (Attribute id msg) -> Tab id msg
+buildTab config attributes =
+    Tab (TabsInternal.fromList config (List.map (\(Attribute f) -> f) attributes))
+
+
+view :
+    { focusAndSelect : { select : id, focus : Maybe String } -> msg
+    , selected : id
+    , tabs : List (Tab id msg)
+    , tabStyles : Int -> Bool -> List Style
+    , tabListStyles : List Style
+    , containerStyles : List Style
+    , tabListPosition : TabPosition
+    }
+    -> Html msg
+view config =
+    let
+        { tabList, tabPanels } =
+            TabsInternal.views
```

Comment:
This is maybe a terrible suggestion? so definitely take it with a grain of salt.

But I'm wondering if it would make sense to return ` { tabList : Html msg, tabPanels : Html msg }` directly.

This way:
- we wouldn't need `containerStyles`
- we wouldn't need `tabListPosition`

and using the carousel would still be really flexible.

Just an idea!

Reply:
I... actually like it? I think it'd be good for this Carousel to be very customizable, since we're probably going to need it in different forms in very different places